docs/pages/build/internal-distribution.md

```diff
@@ -88,28 +80,26 @@ You can register new devices at any time, but builds that were created before th
 
 The next step is to generate or update the provisioning profile. When you proceed to running a build, you will be guided through this process.
 
-> **Are you using manual local credentials?** Make sure to point your `credentials.json` to an ad hoc <!-- or enterprise -->provisioning profile that you generate through the Apple Developer portal. Beware that EAS CLI does only a limited validation of your local credentials, and you will have to handle device UDID registration manually. Read more about [using local credentials](/app-signing/local-credentials.md).
+#### Setting up enterprise provisioning
 
-## 3. Run a build with the internal build profile
+If you plan on using enterprise provisioning, please sign in to the account with [Apple Developer Enterprise Program membership](https://developer.apple.com/programs/enterprise/). You probably don't have this, and it's expensive ($299 USD per year) and takes time to acquire, so you will likely be using ad hoc provisioning &mdash; this works on any normal paid Apple developer account.
 
-Now that we have set up our build profile and app signing, running a build for internal distribution is just like any other build.
+If you do have an Apple enterprise account, this makes internal distribution much easier for users who want to install your app for the first time. Once they install the profile to their device they can access the app right away. One limitation of using an enterprise provisioning profile is that you will need to have a distinct bundle identifier from the one that you use to publish your app to the App Store. We recommend setting your bundle identifier for internal distribution and committing that change on another branch. After that, whenever you want to create a preview branch you can check out that branch and rebase against the branch you'd like to create a build for.
```

Comment:
using app.config.json to dynamically switch between identifiers would a better option for managed.
for generic it's probably easier to use separate branches as we don't support projects with multiple bundle identifiers

Also, you are assuming here that the developer is using an enterprise account for test builds and they distribute their app via app store which in most cases is not true. So I would suggest to preface part about separate branches with `If you distribute your app via app store`